kiwi/system/setup.py

```diff
@@ -87,6 +88,42 @@ def __init__(self, xml_state: XMLState, root_dir: str):
         self._preferences_lookup()
         self._oemconfig_lookup()
 
+    def setup_registry_import(self) -> None:
+        """
+        Fetch container(s) and activate systemd unit to load
+        fetched containers during boot
+        """
+        for container in self.xml_state.get_containers():
```

Comment:
@Conan-Kudo What is your proposal for documentation and what the user is expected to do if they need a service that runs after the container load?

I understand the "better logging" argument. That said if I build an image that includes, let's say Rancher, which has some 60+ containers and I want to run additional setup tasks when all the application containers are installed I end up having to create a `After: A_LIST_OF_60+_SERVICES` unit file. I don't see how we can consider that to be user friendly. Certainly the logging can be solved in a satisfactory way in some other fashion. systemd captures what an executable that is invoked via a unit file outputs, as such some well placed "echo" statements in a generated shell script should provide reasonable debugging information.

Reply:
I think the way it is now offers a proper systemd journal and status information via `systemctl status -l kiwi_containers` and/or `journalctl -u kiwi_containers` or for long running loads  `journalctl -u kiwi_containers -f`

Reply:
If we're going to have dependency information, it would have to be in the XML description anyway. Additionally, Podman already supports being able to [define groups of containers with dependencies](https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html) too with its quadlet functionality.